libgalois/include/katana/Properties.h

```diff
@@ -527,52 +527,53 @@ class VectorRef {
     const T* begin() const { return const_iterator(&data_[0]); }
     T* end() { return iterator(&data_[size_]); }
     const T* end() const { return const_iterator(&data_[size_]); }
+
   private:
     T* data_;
     size_t size_;
 };
 
 
 template <typename T, size_t N>
-class VectorPropertyView {
+class ArrayPropertyView {
   public:
-    using value_type = VectorRef<T>;
-    using reference = VectorRef<T>&;
-    using const_reference = const VectorRef<T>&;
+    using value_type = ArrayRef<T>;
+    using reference = ArrayRef<T>&;
+    using const_reference = const ArrayRef<T>&;
```

Comment:
ArrayRef is itself a reference. I would expect these to read something like (and these may be incorrect c++):
```c++
using value_type = ArrayRef<const T>;
using reference = ArrayRef<T>;
using const_reference =  ArrayRef<const T>;
```
Note that operator= doesn't make sense on the ArrayRef type, so returning a reference wouldn't be productive